airflow/callbacks/callback_requests.py

```diff
@@ -34,13 +35,21 @@ def __init__(self, full_filepath: str, msg: Optional[str] = None):
         self.msg = msg
 
     def __eq__(self, other):
-        if isinstance(other, CallbackRequest):
+        if isinstance(other, self.__class__):
             return self.__dict__ == other.__dict__
         return False
```

Comment:
This should be changed to `return NotImplemented`. Not strictly necessary for Airflow’s particular use case, but this is a best practice.

Reply:
ok, changed